L1Trigger/L1CaloTrigger/plugins/Phase2L1CaloPFClusterEmulator.h

```diff
@@ -0,0 +1,278 @@
+#ifndef _PHASE_2_L1_CALO_PFCLUSTER_EMULATOR_H_
+#define _PHASE_2_L1_CALO_PFCLUSTER_EMULATOR_H_
+
+#include <cstdlib>
+
+//    eta:  0  1  2  3  4   5  6  7  8   9 10 11 12  13 14 15 16  17 18 19 20
+// 0             |                                                     |
+// 1             |                                                     |
+//               |-----------------------------------------------------|
+// 2             |                                                     |
+// 3             |                                                     |
+// 4             |                                                     |
+// 5             |                                                     |
+//               | ----------------------------------------------------|
+// 6             |                                                     |
+// 7             |                                                     |
+//
+// 8 PFclusters are created in one 21x8 (2+17+2 x 2+4+2)
+
+typedef struct {
+float et ;
+int eta ;
+int phi ;
+} GCTpfcluster_t ;
+
+typedef struct {
+GCTpfcluster_t GCTpfclusters[8] ;
+} GCTPfcluster_t ;
+
+typedef struct {
+float et ;
+int eta ;
+int phi ;
+} GCTint_t ;
+
+typedef struct {
+   GCTint_t t[8];
+} gctEtaStrip_t ;
+
+typedef struct {
+   GCTint_t p[19];
+} gctEtaStripPeak_t ;
+
+typedef struct {
+   gctEtaStrip_t s0;
+   gctEtaStrip_t s1;
+   gctEtaStrip_t s2;
+   gctEtaStrip_t s3;
+   gctEtaStrip_t s4;
+   gctEtaStrip_t s5;
+   gctEtaStrip_t s6;
+   gctEtaStrip_t s7;
+   gctEtaStrip_t s8;
+   gctEtaStrip_t s9;
+   gctEtaStrip_t s10;
+   gctEtaStrip_t s11;
+   gctEtaStrip_t s12;
+   gctEtaStrip_t s13;
+   gctEtaStrip_t s14;
+   gctEtaStrip_t s15;
+   gctEtaStrip_t s16;
+   gctEtaStrip_t s17;
+   gctEtaStrip_t s18;
+   gctEtaStrip_t s19;
+   gctEtaStrip_t s20;
+} region_t;
```

Comment:
These struct definitions are all at global scope, which is very probably not going to be allowed. At the least, these should be associated with a namespace.